evm_arithmetization/src/cpu/kernel/constants/mod.rs

```diff
@@ -397,7 +397,9 @@ pub mod cancun_constants {
         hex!("000000000000000000000000000000001666c54b0a32529503432fcae0181b4bef79de09fc63671fda5ed1ba9bfa07899495346f3d7ac9cd23048ef30d0a154f"), // y_im
     ];
 
-    pub const HISTORY_BUFFER_LENGTH: (&str, u64) = ("HISTORY_BUFFER_LENGTH", 8191);
+    pub const HISTORY_BUFFER_LENGTH_VALUE: u64 = 8191;
```

Comment:
You can define it as a `U256` here so that you don't need to convert it above nor on the decoder side.